kola/harness.go

```diff
@@ -577,6 +577,9 @@ func architecture(pltfrm string) string {
 	if pltfrm == "aws" && AWSOptions.Board != "" {
 		nativeArch = boardToArch(AWSOptions.Board)
 	}
+	if pltfrm == "azure" && AzureOptions.Board != "" {
```

Comment:
Oh, the saving of keystrokes. Extending lifetime of our dear keyboards. Heh.

(No need to change anything, just being amused here. :) )

Reply:
I think it's still too long, the `r` is a waste of space